sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/_tests_/unit/e-cert-full-time-integration.service.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { createMock } from "@golevelup/ts-jest";
+import { SshService } from "@sims/integrations/services";
+import { DisbursementValueType, RelationshipStatus } from "@sims/sims-db";
+import { ConfigService } from "@sims/utilities/config";
+import {
+  getGenderCode,
+  getMaritalStatusCode,
+  getPPDFlag,
+} from "@sims/utilities/esdc-utils";
+import { ECertFullTimeFileFooter } from "../../e-cert-files/e-cert-file-footer";
+import { ECertFullTimeFileHeader } from "../../e-cert-files/e-cert-file-header";
+import { ECertFullTimeIntegrationService } from "../../e-cert-full-time.integration.service";
+import { RecordTypeCodes } from "../../../models/e-cert-integration-model";
+
+describe("ECertFullTimeIntegrationService-createRequestContent", () => {
+  let eCertFullTimeIntegrationService: ECertFullTimeIntegrationService;
+  let configService: ConfigService;
+  let sshService: SshService;
+
+  beforeAll(() => {
+    configService = createMock<ConfigService>();
+    sshService = createMock<SshService>();
+    const eCertFullTimeFileHeader = new ECertFullTimeFileHeader();
+    const eCertFullTimeFileFooter = new ECertFullTimeFileFooter();
+    eCertFullTimeIntegrationService = new ECertFullTimeIntegrationService(
+      eCertFullTimeFileHeader,
+      eCertFullTimeFileFooter,
+      configService,
+      sshService,
+    );
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("Should create content for full time E-CERT file when requested providing E-CERT records.", async () => {
+    // Arrange
+    const ecertRecords = [
+      {
+        sin: "445966120",
+        stopFullTimeBCFunding: false,
```

Comment:
@guru-aot Is this property needed? I don't it as a part of the interface `ECertRecord`.

Reply:
you are right @sh16011993, i will remove it.

Reply:
you are right @sh16011993